supervisor/host/network.py

```diff
@@ -100,6 +99,28 @@ def get(self, inet_name: str) -> Interface:
             self.sys_dbus.network.interfaces[inet_name]
         )
 
+    def _check_dbus(self):
+        """Check available dbus connection."""
+        if not self.sys_dbus.network.is_connected:
+            raise HostNotSupportedError(
+                "No systemd D-Bus connection available", _LOGGER.error
+            )
+
+    async def load(self):
+        """Load network information and reapply defaults over dbus."""
+        self._check_dbus()
```

Comment:
That is the old style. Create and use somethings like `@Job(conditions=[JobCondition.HOST_NETWORK])` (pseudo code) - i.e. https://github.com/home-assistant/supervisor/blob/main/supervisor/homeassistant/core.py#L468